<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0,206,209)</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>610</y>
     <width>641</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonLayout">
    <property name="spacing">
     <number>7</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <widget class="QToolButton" name="OneKeyButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>一键签到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="InputButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>录入信息</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="InfoButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="AttendButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>考勤管理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="JITpiclabel">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>130</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Infopiclabel">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>130</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="simlabel">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>200</y>
     <width>81</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Adobe 黑体 Std R&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>400</y>
     <width>541</width>
     <height>271</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>信息显示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="Infopic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>工号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>90</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>年龄：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>部门：</string>
    </property>
   </widget>
   <widget class="QLabel" name="infoIDlabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoNamelabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoAgelabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoGenderlabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="infoApartlabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font: 28pt &quot;Adobe 楷体 Std R&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>人脸检索考勤系统</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGraphicsView" name="CameragraphicsView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>640</width>
     <height>480</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>30</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>541</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>实时显示</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>groupBox_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>JITpiclabel</zorder>
  <zorder>Infopiclabel</zorder>
  <zorder>simlabel</zorder>
  <zorder>groupBox</zorder>
  <zorder>label_4</zorder>
  <zorder>CameragraphicsView</zorder>
  <zorder>exitButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
